### ISSUE TITLE
Support running kafka-ui on subset of topics to get around auth errors

### ISSUE THREAD

**Issue by MaksimMyshkin** [provectus/kafka-ui#1452]
### Is your proposal related to a problem?
In our company we have kafka cluster common for all teams but it does not have convenient UI for our needs. Our team tried to deploy kafka-ui with our account that has only access to our topics but it failed with access error. It would be useful to have possibility to run kafka-ui on subset of topics to which account have the needed rights.

Addition question on described problem: Which rights kafka-ui needs to run correctly? Can't find it on project page.

### Describe the solution you'd like
The easiest solution (from my perspective) is to provide the config with a list of topics to which specified account has access.

As alternative kafka-ui can just skip authorization error on topics.

**Comment by Haarolean** [comment]
Hey, thanks for reaching out.

This is for sure to be done via ignoring permission issues. We'll get this done in future releases :)

**Comment by Haarolean** [comment]
#928 #753 #482

**Comment by vdanylenko** [comment]
@MaksimMyshkin Currently kafka-ui needs WRITE and DESCRIBE_CONFIGS permissions on a topic to display it correctly. If the account used to configure kafka-ui  has some access to any topic where DESCRIBE_CONFIGS is not allowed no topics is displayed at all. You might want to add DESCRIBE_CONFIGS  to all your topics as a workaround.

**Comment by MaksimMyshkin** [comment]
Thank you for information. Unfortunately our team can't get access to all topics in the cluster (only to our topics). 

Also I dug a little and found out that errors is appearing on creation of AdminClient. It is even before getting information about topics. Is this code a reason why rights to all topics required? Looks like AdminClient requires wider permissions than it is needed for simpler scenarios (testing by producing and reading topics messages for example).

**Comment by vdanylenko** [comment]
@MaksimMyshkin Sorry, looks like I didn't make myself clear enough. It was not about all topics in the cluster, only about "your" topics. If you have any access to a topic then WRITE and DESCRIBE_CONFIGS should be allowed for that topic.

**Comment by maratgaliev** [comment]
@vdanylenko hey! Any updates here? What's the status on this issue?

**Comment by MaksimMyshkin** [comment]
I afraid permissions that @vdanylenko described is not enough to work with topics from kafka-ui. At the moment I think that problem due to using AdminClient and I don't known how to work around it right. I just trying to find a reason to the problem that I (or someone else) can help to fix it sooner(

**Comment by Haarolean** [comment]
@MaksimMyshkin hey, wanna hop on our discord? We can try to investigate this. The link is available in README.

**Comment by moinmoin** [comment]
Using Kafka-UI in a multi-tenant scenario is not working currently. Main reason is, that there are no clusterwide permissions assigned to users. So referring to this table the following permissions are granted:
https://docs.confluent.io/platform/current/kafka/authorization.html

Topics:
Describe, DescribeConfigs, Read and if needed Write (single topic oder wildcard) ; note: read implies describe
Group:
Describe
Transactional ID:
Describe

The following data is available then (without cluster permissions) in a different tool:
![topics](https://user-images.githubusercontent.com/3028772/173685181-2454d5ee-9fab-4f02-b665-a0f938894583.png)
![brokers](https://user-images.githubusercontent.com/3028772/173693478-f97cff0c-4a56-4686-aa3a-b531ac9c4b11.png)
![overview](https://user-images.githubusercontent.com/3028772/173693494-6ad50a3d-a5e0-4e4a-81b2-595dfbb99184.jpg)

**Comment by Haarolean** [comment]
@moinmoin thanks for the feedback!

**Comment by allanpaiste** [comment]
Faced with similar issue where same proposed solution would be good enough for our use (relatively small tradeoff in situations where the list of topics is anyways known and does not grow with such speed that would cause the manual maintenance on the list to be a problem).

**Comment by TaylorSMarks** [comment]
> The following data is available then (without cluster permissions) in a different tool:

@moinmoin - Don't hold out on us - what tool is this?

**Comment by moinmoin** [comment]
@TaylorSMarks This is the https://www.conduktor.io/download fat client.

**Comment by TaylorSMarks** [comment]
I have opened a small PR which fixes the root issue (you lack permissions to run describe cluster), although I accomplished it by just ignoring the exception you get instead of going into a crash loop. I don't think adding the feature to be able to run on a subset of topics would actually fix the root issue, since the Admin Client would still fail to create with an auth exception.

Everything runs pretty ok with the change.

Here's the PR:
https://github.com/provectus/kafka-ui/pull/3325

The directions for opening a PR said I should request reviewers and add myself as the owner for the PR, but I lack permissions to do either of those things. Here's hoping the right people see, review, approve, and merge the PR...
